functional-tests/src/test/java/org/infinispan/online/service/endpoint/EndpointTester.java

```diff
@@ -7,7 +7,5 @@
 
    void testBasicEndpointCapabilities(URL urlToService);
 
-   void testPutPerformance(URL urlToService, long timeout, TimeUnit timeUnit);
```

Comment:
Why are we removing this?

Reply:
This test have the same semantics as `should_put_entries_until_first_one_gets_evicted`. So I decided to remove is "as a duplicate".